common/spinner.py

```diff
@@ -25,7 +25,10 @@ def update(self, spinner_text: str):
         pass
 
   def update_progress(self, cur: int, total: int):
-    self.update(str(round(100 * cur / total)))
+    # Always floor progress bars because they indicate fully completed chunks 
```

Comment:
no need for the comment here

Reply:
sorry, i've had to have this conversation a surprising amount of times so i suppose i was trying to head it off at the pass